server/config/index.js

```diff
@@ -28,17 +28,19 @@ if (process.env.PLAYGROUND_VARIABLES_FILE) {
   }
 }
 
+const topLevelGraphqlPath = '/graphql'
 const port = process.env.HTTP_PORT || '8000'
 
 const config = {
   server: {
+    apiPath: topLevelGraphqlPath,
```

Comment:
Can you rename this to `graphqlEndpoint` and move it under the graphqlConfig object?